tests/test_auth.py

```diff
@@ -48,32 +61,38 @@ def test_endpoint(auth_mock):
     assert auth_mock._endpoint() == "https://api.up42.abc"
 
 
-def test_get_token(auth_mock):
-    auth_mock._get_token()
-    assert auth_mock.token == TOKEN
+@pytest.mark.parametrize("authentication_fixture", ("auth_mock", "auth_account_mock"))
```

Comment:
Actually we need to change in auth_fixtures to 
```
@pytest.fixture(params=["project", "account"])
def auth(request, auth_project_mock, auth_account_mock):
    mocks = {"project": auth_project_mock, "account": auth_account_mock}
    return mocks[request.param]
```
to run all tests with both mocks if they don't specify which kind of auth type they need

Reply:
And the same below